pwiz_tools/Skyline/TestA/CommandLineTest.cs

```diff
@@ -1095,7 +1095,7 @@ public static int CountErrors(string searchSpace, bool allowUnlocalized = false)
             return count;
         }
         
-        [TestMethod]
+        //[TestMethod]

```

Comment:
Obviously, don't merge this to master.

Reply:
No longer commented out with the latest push.